README.md

```diff
@@ -196,16 +205,5 @@ See [alternatives-considered.md](./alternatives-considered.md)
 
 ## Open Questions
 
-1. Does the app authenticate in the the postMessage body? Should the app pass the access_token in the postmessage to authenticate itself? 
-
-* Currently, we don’t have this in the above, because of the concern around bringing the access_token to the client in javascript, and because the host having launched the app is what authenticates the app.
-Should there be something that is exchanged between the app and the host to authenticate the app at all? Or is the host having launched the app enough authorization for the app? 
-
-2. Does access expire with SMART access_tokens?
-
-3. How does the app know what messageTypes are supported? 
-There are two aspects to this:
-* Does the container at all support messaging?
-* What messages are supported in this context?
 
-4. Does (3) above need to be an initial handshake postMessage? Do scopes in access_token already meet this need? Or do we need something like added details in a well-known/smart-configuration.json / documentation?
+* Does (3) above need to be an initial handshake postMessage? Do scopes in access_token already meet this need? Or do we need something like added details in a well-known/smart-configuration.json / documentation?
```

Comment:
The `(3) above` no longer makes sense (since it has been removed).  Maybe re-word this open question to include the relevant context.

Reply:
Updated to resolve the dangling reference to (3).